src/Phug/Reader.php

```diff
@@ -257,7 +261,7 @@ public function getLength()
      */
     public function hasLength()
     {
-        return $this->getLength() > 0;
+        return $this->input !== '';
```

Comment:
`$this->input` can be `false` on PHP 5. I guess it's why Travis freezes: https://travis-ci.org/phug-php/reader/jobs/315405734